pkg/querier/querier.go

```diff
@@ -110,6 +110,7 @@ func (q *V3ioQuerier) selectQry(
 			return
 		}
 
+		// Bug: TSDB-37 -> TODO: Create result set while taking in account the step. If step is greater than partition interval it should return correct number of entries
```

Comment:
It's not really clear why `TSDB-37` is related to steps. The ticket itself only mentions partitions. Also not really related to this PR 🙂